ocaml/xapi/memory_check.ml

```diff
@@ -108,12 +108,15 @@ let vm_compute_used_memory ~__context policy vm_ref =
    		request is more than the total free.
 *)
 type host_memory_summary = {
+  host_maximum_guest_memory_bytes: int64
   (** The maximum amount of memory that guests can use on this host. *)
-  host_maximum_guest_memory_bytes: int64;
+  ;
+  resident: API.ref_VM list
   (** list of VMs which have a domain running here *)
-  resident: API.ref_VM list;
+  ;
+  scheduled: API.ref_VM list
```

Comment:
What happened here? I would've expected the `;` to go on same line as one of `resident`/`scheduled`

Reply:
The field separator is `;`, so the field and the comment must not be separated by `;`

I can format it as ocamlformat would:
```
let ... =
{
; field : type (** doc *)
; field2 : type (** doc *)
)